modules/rostests/apitests/include/apitest.h

```diff
@@ -12,6 +12,10 @@
 #define InvalidPointer ((PVOID)0x5555555555555555ULL)
 // #define InvalidPointer ((PVOID)0x0123456789ABCDEFULL)
 
+// Magic pointers come from KUSER_SHARED_DATA; needed to get true NT version on Windows 8.x+
```

Comment:
Nit for consistency with other comments in this file:
```suggestion
/* Magic pointers come from KUSER_SHARED_DATA; needed to get true NT version on Windows 8+ */
```

Reply:
The comment just above is `//`. `//` also looks better for single lines IMHO. There is a reason both comment types exist.